woocommerce/woocommerce-ios
# Remove product SKU scanner feature switch to enable for all users

Part of #2407 


## Description


After confirming with the product p1683102647126129/1683097283.360639-slack-C03L1NF1EA3, we're removing the feature switch for the product SKU scanner feature that has been launched behind a feature switch over a year ago. The feature switch is pretty hidden, and the iOS 16 camera still doesn't support barcode scanning right now.

In addition to removing the feature switch code, I also fixed a performance warning from Xcode from starting the camera when testing on a device. I followed the fix for the same warning in https://github.com/wordpress-mobile/WordPress-iOS/pull/19636 and the warning didn't appear anymore.

## Testing instructions


Prerequisite: please test on a physical device with a camera

- Log in if needed
- Go to the Products tab
- Create a simple product if the store has no simple products
- Tap on a simple product
- Tap on `Inventory` row --> there should be a camera icon at the trailing edge
- Tap on the camera icon --> it should prompt for permission if it's started for the first time
- Tap to accept the permission
- Try scanning a barcode --> it should be filled to the product SKU field

---
- [x] @jaclync tested the camera permission disabled flow 
- [x] @jaclync tested that Settings > Experimental Features has the same states after the PR changes 

## Screenshots


Product SKU

<img src="https://user-images.githubusercontent.com/1945542/236136914-f14a3370-cd36-4387-95f2-c9d564d9fb17.png" width="300" />

before | after
-- | --
![Simulator Screen Shot - iPhone 14 - 2023-05-04 at 14 46 14](https://user-images.githubusercontent.com/1945542/236136879-d0f3da09-56f2-437b-9a11-93e13f6625ad.png) | ![Simulator Screen Shot - iPhone 14 - 2023-05-04 at 15 21 34](https://user-images.githubusercontent.com/1945542/236136922-222667c7-0949-4246-bd0d-0be2355c5ef5.png)


---
